features/admin/maps/admin_configures_maps.feature

```diff
@@ -0,0 +1,21 @@
+Feature: Admin configures maps
+
+  In order to use Google Maps APIs and make it possible for users to find listings by location
+  As an admin
+  I want to configure Google Maps API key
+
+  Background:
+    Given there are following users:
+      | person            | given_name | family_name | email               | membership_created_at     |
+      | manager           | matti      | manager     | manager@example.com | 2014-03-01 00:12:35 +0200 |
+      | kassi_testperson1 | john       | doe         | test2@example.com   | 2013-03-01 00:12:35 +0200 |
+      | kassi_testperson2 | jane       | doe         | test1@example.com   | 2012-03-01 00:00:00 +0200 |
+    And I am logged in as "manager"
+    And "manager" has admin rights in community "test"
+    And I am on the maps admin page
+
+  Scenario: Admin adds a google maps api key
+  When I add "fookey" to the Google Maps API key field
+  And I press submit
+  And I refresh the page
+  Then I should see "fookey" in the Google Maps API key field
```

Comment:
I have a personal problem with Cucumber tests. 😟  They are slow and quite verbose. We haven't agreed anything, but personally I would even suggest that we don't add any new Cucumber tests.

Would it make sense to test this with RSpec instead? I don't think the browser interaction brings any additional value to this test, so maybe we could test this without browser, that is, with RSpec? Or, of course, one option is not to add test at all if the feature is simple and test doesn't bring any value...

What do you think?